sql/src/test/com/github/sadikovi/spark/riff/SparkFiltersSuite.scala

```diff
@@ -85,8 +135,8 @@ class SparkFiltersSuite extends UnitTestSuite {
   test("convert And into tree") {
     Filters.createRiffFilter(Some(And(EqualTo("col1", "a"), GreaterThan("col2", 1)))) should be (
       and(eqt("col1", "a"), gt("col2", 1)))
-    Filters.createRiffFilter(Some(And(IsNotNull("col1"), EqualTo("col1", 1)))) should be (
-      and(FilterApi.not(nvl("col1")), eqt("col1", 1)))
+    Filters.createRiffFilter(Some(And(IsNotNull("col2"), EqualTo("col1", 1)))) should be (
```

Comment:
Since we added rule to collapse such filters, I had to change column name instead of modifying expected filter.